CHANGELOG.rst

```diff
@@ -1,5 +1,47 @@
 .. _changelog:
 
+0.49.0
+------
+
+The release contains bug fixes to renku core service related to project migration.
+
+This release also contains initial support for next generation 'Renku 1.0' functionality. However,
```

Comment:
Here and everywhere else:

```suggestion
This release also contains initial support for next generation 'Renku 2.0' functionality. However,
```

Reply:
I will make a PR to update Renku 1.0 references for the whole changelog